real_estate/models/estate_property.py

```diff
@@ -0,0 +1,143 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+import datetime
+
+class EstateProperty(models.Model):
+
+    _name = 'estate.property',
+    _description = 'Real Estate Property'
+
+    name = fields.Char(
+        string='Property Name', 
```

Comment:
```suggestion
        string='Name',
```
Don't you think Model itself defines it's a property? Let me know your opinion on it 😄 

Also I see a bad trailing space at the end of line. It's not a good practice to have a extra space at the end of line

Reply:
will work on it :sweat_smile: